lib/v2/baidu/tieba/search.js

```diff
@@ -0,0 +1,64 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const iconv = require('iconv-lite');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const { art } = require('@/utils/render');
+const path = require('path');
+const { URLSearchParams } = require('url');
```

Comment:
```suggestion
```
https://nodejs.org/api/url.html#class-urlsearchparams
URLSearchParams is also available via the global object which does not require importing.